tests/function-controller/cmd/main.go

```diff
@@ -2,57 +2,70 @@ package main
 
 import (
 	"math/rand"
-	"testing"
-	"time"
-
-	"k8s.io/client-go/rest"
-
-	"github.com/sirupsen/logrus"
+	"os"
 
 	"github.com/kyma-project/kyma/tests/function-controller/pkg/step"
+	"github.com/kyma-project/kyma/tests/function-controller/testsuite"
 	"github.com/kyma-project/kyma/tests/function-controller/testsuite/scenarios"
-
-	"github.com/onsi/gomega"
+	"github.com/sirupsen/logrus"
 	"github.com/vrischmann/envconfig"
-	_ "k8s.io/client-go/plugin/pkg/client/auth"
-	controllerruntime "sigs.k8s.io/controller-runtime"
+	"k8s.io/client-go/rest"
 
-	"github.com/kyma-project/kyma/tests/function-controller/testsuite"
-)
+	"time"
 
-type config struct {
-	Test testsuite.Config
-}
+	controllerruntime "sigs.k8s.io/controller-runtime"
+)
 
-func TestRuntimes(t *testing.T) {
-	runTests(t, scenarios.FunctionTestStep)
+var availableScenarios = map[string][]testSuite{
+	"serverless-integration": {scenarios.SimpleFunctionTest},
+	"kyma-integration":       {scenarios.FunctionTestStep, scenarios.GitopsSteps},
 }
 
-func TestGitSourcesFunctions(t *testing.T) {
-	runTests(t, scenarios.GitopsSteps)
+type config struct {
+	Test testsuite.Config
 }
 
-type testSuite func(*rest.Config, testsuite.Config, *logrus.Entry) (step.Step, error)
+func main() {
+	logf := logrus.New()
+	logf.SetFormatter(&logrus.TextFormatter{})
+	logf.SetReportCaller(false)
 
-func runTests(t *testing.T, testFunc testSuite) {
-	rand.Seed(time.Now().UnixNano())
```

Comment:
Are you sure that there are no things in the whole tests that would need this? I remember that we generated namespace name, right now we basing that on current test time, but maybe we have similar logic somewhere else?